packages/lxd_service/lib/src/terminal.dart

```diff
@@ -8,7 +8,7 @@ import 'package:meta/meta.dart';
 mixin LxdTerminalService on LxdClient {
   Future<LxdTerminal> execTerminal(LxdInstanceId id) async {
     final instance = await getInstance(id);
-    final user = instance.config['user.name'] ?? 'root';
+    final user = instance.config['user.workshops.name'] ?? 'root';
```

Comment:
I just realized this affects existing instances, i.e. launches the terminal as root instead of the user created in the beginning

Reply:
yes, let's leave the fallback for now. we can throw it out sometime in the distant future :)

Reply:
`user.username` looked silly so I went with `user.name` but now with the prefix, `user.workshops.username` could work fine. now we'd have a chance to name it if we want to be more clear... what do you think?